identity-iota/src/diff/mod.rs

```diff
@@ -1,11 +1,4 @@
 // Copyright 2020-2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-pub use self::diff_iota_document::DiffIotaDocument;
-pub use self::diff_iota_document_metadata::DiffIotaDocumentMetadata;
-pub use self::diff_message::DiffMessage;
-
-mod diff_iota_did;
-mod diff_iota_document;
-mod diff_iota_document_metadata;
-mod diff_message;
+pub mod diff_message;
```

Comment:
This folder can be removed if we just move the trait implementation to where `TangleRef` is defined.